gaidds/React-crm-project
# Create modal component

- [x] prepopulate edit

<hr/>

## What is added:

I created the infrastructure of a DynamicModal component which requires the page name and the mode (edit/add) and then renders the relevant form (Deals, Users, Accounts, Contacts).

In this branch I have only completed the deals forms since Deals pages are our part to complete in this sprint.

<hr/>

## How to test:

Please create and edit your deals and check wether you get the relevant erros on the forms, and wether when you edit the prepopulation the form works properly.

<hr/>

**Have not worked on the styling of this component**
